apps/model_eval_viz/app.py

```diff
@@ -0,0 +1,41 @@
+import hvplot.pandas
+import numpy as np
+import panel as pn
+import pandas as pd
```

Comment:
Swap these imports so they are in alphabetical order.

We will be adding the `ruff` linter to our toolbox soon, which will take care of this in the future.